docs/kyma-environment-broker/02-01-architecture.md

```diff
@@ -5,6 +5,13 @@ The diagram and steps describe the Kyma Environment Broker (KEB) workflow and th
 ![KEB diagram](./assets/keb-architecture.svg)
```

Comment:
Please adjust connectors (1a and 1c) in the diagram according to the [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/02-diagrams.md#examples).